cscore/src/main/native/include/cscore_raw_cv.h

```diff
@@ -9,6 +9,8 @@
 #error "Cannot include both cscore_cv.h and cscore_raw_cv.h in the same file"
 #endif
 
+#include <opencv2/core/core.hpp>
```

Comment:
We explicitly didn't have raw including opencv. Why is this needed?

Reply:
Hmm, but this header definitely doesn't follow style guidelines then.  And doesn't the user have control of what include directories the compiler is using?

Reply:
The design of this was to not allow standalone, but require including opencv manually before hand. I'd seen some opencv distributions where there was no `opencv2` folder, and instead you just needed `core/core.hpp`

Reply:
Ok, removed.